pegasus/src/homepage.rb

```diff
@@ -555,10 +555,11 @@ def self.show_courses_banner(request)
 
   def self.get_dance_stars
     stars = [
-      "Katy Perry", "Lil Nas X (ft. Billy Ray Cyrus)", "Jonas Brothers", "Panic! At The Disco",
-      "Shawn Mendes", "Nicki Minaj", "KIDZ BOP", "Pedro Capó", "Francesco Gabbani", "Sia",
-      "A-ha", "Ariana Grande", "Avicii and Aloe Blacc", "Calvin Harris",
-      "Carly Rae Jepsen", "Ciara", "Coldplay", "Ed Sheeran", "Imagine Dragons",
+      "Beyoncé", "Harry Styles", "Lizzo", "Post Malone", "Disney\'s \"Encanto\"", "Nicky Youre",
+      "Katy Perry", "Lil Nas X", "Jonas Brothers", "Panic! At The Disco",
+      "Shawn Mendes", "Nicki Minaj", "Pedro Capó", "Francesco Gabbani", "Sia",
+      "Ariana Grande", "Avicii and Aloe Blacc", "Calvin Harris",
+      "Carly Rae Jepsen", "Coldplay", "Ed Sheeran", "Imagine Dragons",
       "J Balvin and Willy William", "Justin Bieber", "Keith Urban", "Lady Gaga",
       "Los del Río", "Madonna", "Mark Ronson (ft. Bruno Mars)", "MC Hammer",
       "Miley Cyrus", "Selena Gomez", "The Weeknd", "Yolanda Be Cool"
```

Comment:
Just below this array, I see `DCDO.get("hoc2019_dance_stars", stars)`. Do you know what the `hoc2019_dance_stars` is used for? Does it need to be updated for this year?

Reply:
I don't see the flag in AWS (used https://docs.google.com/document/d/1l5ISiVT329Y9uKLZYPmrSvu63O3DrB2iVX4xKsfeQhA/edit# as a reference), so it looks safe to remove! Thanks